pages/common/git-commit.md

```diff
@@ -15,6 +15,10 @@
 
 `git commit -a -m "{{message}}"`
 
+- Commit staged files and sign them with the GPG key defined in the config:
+
+`git commit -S -m "{{message}}"`
```

Comment:
This is not a common command, could we use this?
```suggestion
`git commit --gpg-sign -m "{{message}}"`
```

Reply:
I think it's implied in the description that `-S` signs with a GPG key. Also, `s` for `sign` is pretty obvious. Also if you're using `-S` instead of auto-signing it's as common as `-m`.

Reply:
Exactly, I'd assume most that use GPG signing use the auto sign in .gitconfig

If I didn't then all of my commands would use -S, making it just as used as -m

Reply:
@navarroaxel anything to add to this or resolve?

Reply:
I suggest either using the long form or using `[S]` in the description.

Reply:
I think when you're actually using `-S` it would be very impractical to use the long form, so I'll go with `[S]ign`